src/types/index.ts

```diff
@@ -17,7 +21,7 @@ export interface TemplateLintResult {
 
 export interface TemplateLintMessage {
   rule: string;
-  severity: Severity;
+  severity: 0 | 1 | 2;
```

Comment:
Can't we use the enum somehow (vs hardcoding / duplication)?

Reply:
I'm not sure. Essentially we'd want the union of the enum values expressed as a type.

Reply:
I think I'll leave for now, as this is likely going away when we convert `ember-template-lint` to TS.